### ISSUE TITLE
Observables to use

### ISSUE THREAD

**Issue by tmcclintock** [LSSTDESC/clmassmod#5]
Clusters have various weak lensing observables associated with them that different people use. You can use Kappa maps, gamma(theta), DeltaSigma and other signals. It might be a good idea to have CLMM work on one of these (which might be what Doug had already, not sure) as opposed to "weak lensing" in general.

**Comment by aimalz** [comment]
Good idea!  I added this to [the wishlist notebook in the refactoring branch](https://github.com/LSSTDESC/clmassmod/blob/refactoring/docs/wishlist.ipynb) to the best of my understanding.
